app/sanitizers/rendered_markdown_scrubber.rb

```diff
@@ -39,7 +40,7 @@ def scrub_attributes(node)
 
   def scrub_valid_attributes(node)
     node.attributes.each_value do |attribute|
-      attribute.value = attribute.value.gsub(/\{%|%\}/, "")
+      attribute.value = attribute.value.gsub(LIQUID_TAG_SYNTAX_REGEX, "")
```

Comment:
does `String#remove( regex )` fit here in place of `gsub(regex, '')`?

Reply:
Huh that's funny, I didn't know that was a method. Wonder why it's not in the Ruby doc: https://ruby-doc.org/core-2.7.2/String.html

Reply:
it's an ActiveSupport core extension to String (so not "pure" ruby, only available in non-core docs). Again, it's only an "intention revealing" convenience wrapper over gsub so totally unneeded.